api/v1alpha1/resource_types.go

```diff
@@ -84,23 +82,22 @@ func (in *Resource) SetConditions(conditions []metav1.Condition) {
 	in.Status.Conditions = conditions
 }
 
-// GetRequeueAfter returns the duration after which the Resource must be
-// reconciled again.
+// GetRequeueAfter returns the duration after which the Resource should be reconciled.
 func (in Resource) GetRequeueAfter() time.Duration {
 	return in.Spec.Interval.Duration
 }
 
-// GetReferencePath returns the component reference path for the resource
+// GetReferencePath returns the component reference path for the Resource.
 func (in Resource) GetReferencePath() []ocmmetav1.Identity {
 	return in.Spec.SourceRef.ResourceRef.ReferencePath
 }
 
-// GetSnapshotName returns the key for the snapshot produced by the Configuration
+// GetSnapshotDigest returns the digest Resource's associated Snapshot.
```

Comment:
// GetSnapshotDigest returns the latest digest of the Resource's associated Snapshot.